src/Application.cpp

```diff
@@ -590,6 +593,13 @@ SeventvEmotes *Application::getSeventvEmotes()
     return this->seventvEmotes.get();
 }
 
+pronouns::Pronouns *Application::getPronouns()
+{
+    assert(this->pronouns);
```

Comment:
Nit: Some comment about not having an assert for the thread here, like in SeventvBadges:
```cpp
// pronouns::Pronouns handles its own locks, so we don't need to assert that this is called in the GUI thread
```